packages/core/docs/i18n.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story, Preview, API } from '@storybook/addon-docs/blocks';
+import { html, LitElement } from 'lit-element';
+
+<Meta title="Foundation/Internationalization" />
+
+# Internationalization (i18n)
+
+Clarity Core provides an i18n service to change the internal component strings for various region support easily. Clarity Core i18n can be controlled in two different ways.
```

Comment:
`to  change the text inside components to support different languages or easily customize strings inside the web component`

...instead of...?

`change the internal component strings for various region support easily`